src/blocks/TitleParagraphBlock.tsx

```diff
@@ -0,0 +1,71 @@
+import * as React from 'react'
+import styled from 'react-emotion'
+import {
+  ContentWrapper,
+  MOBILE_BP_DOWN,
+  SectionWrapper,
+} from '../components/blockHelpers'
+import { BaseBlockProps, MarkdownHtmlComponent } from './BaseBlockProps'
+
+const AlignableContentWrapperComponent = styled(ContentWrapper)(
+  ({ textPosition }: { textPosition: string }) => ({
```

Comment:
can we use https://github.com/HedvigInsurance/web-next/blob/8e4251f5a82350a0b2ae20a771748b7d6c184858/src/utils/utils/textPosition.ts#L1 here?

Reply:
Oh yea! Didn't know we had this type